presentation/src/main/java/com/sopt/presentation/auth/login/OnboardingRoute.kt

```diff
@@ -0,0 +1,154 @@
+package com.sopt.presentation.auth.login
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.pager.HorizontalPager
+import androidx.compose.foundation.pager.rememberPagerState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.component.button.NoostakBottomButton
+import com.sopt.core.designsystem.theme.NoostakAndroidTheme
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.presentation.R
+
+@Composable
+fun OnboardingRoute(
+    authId: String,
+    navigateToSignUp: (String) -> Unit
+) {
+    OnboardingScreen(
+        onNextButtonClick = { navigateToSignUp(authId) }
+    )
+}
+
+@Composable
+fun OnboardingScreen(
+    onNextButtonClick: () -> Unit
+) {
+    val pages = listOf(
+        BoardingPage(
+            imageRes = R.drawable.ic_onboarding_1,
+            subtitle = stringResource(R.string.tv_onboarding_subtitle_1),
+            title = stringResource(R.string.tv_onboarding_title_1)
+        ),
+        BoardingPage(
+            imageRes = R.drawable.ic_onboarding_2,
+            subtitle = stringResource(R.string.tv_onboarding_subtitle_2),
+            title = stringResource(R.string.tv_onboarding_title_2)
+        ),
+        BoardingPage(
+            imageRes = R.drawable.ic_onboarding_3,
+            subtitle = stringResource(R.string.tv_onboarding_subtitle_3),
+            title = stringResource(R.string.tv_onboarding_title_3)
+        )
+    )
```

Comment:
P1: 바뀌지 않는 객체들이니까 immutableList로 선언해줍시다!! 그리고 해당 변수를 route에 선언해주고 screen으로 넘겨주는거 어떤가요?
```suggestion
    val pages = immutableListOf(
        BoardingPage(
            imageRes = R.drawable.ic_onboarding_1,
            subtitle = stringResource(R.string.tv_onboarding_subtitle_1),
            title = stringResource(R.string.tv_onboarding_title_1)
        ),
        BoardingPage(
            imageRes = R.drawable.ic_onboarding_2,
            subtitle = stringResource(R.string.tv_onboarding_subtitle_2),
            title = stringResource(R.string.tv_onboarding_title_2)
        ),
        BoardingPage(
            imageRes = R.drawable.ic_onboarding_3,
            subtitle = stringResource(R.string.tv_onboarding_subtitle_3),
            title = stringResource(R.string.tv_onboarding_title_3)
        )
    )
```